resources/files/ruby_243/rbconfig/rbconfig-ppc64-redhat-linux.rb

```diff
@@ -0,0 +1,313 @@
+# encoding: ascii-8bit
```

Comment:
This file shouldn't be needed at all - rbconfigs are handled in the puppet-runtime project now, and should be automatically generated.

Reply:
True, however, they appear to still be referenced in the puppet-agent project. I included it for completeness.